lib/recommendation-server.js

```diff
@@ -82,17 +89,36 @@ RecommendationServer.prototype = {
       req.send();
     }
   },
+
+  getResultClass: function(data) {
+    let types = Object.keys(data.enhancements);
+    if (types.indexOf('tld') > -1) {
+      return this.resultClasses.tld;
+    } else if (types.indexOf('wikipedia') > -1) {
+      return this.resultClasses.wikipedia;
+    }
+    return null;
```

Comment:
I don't think that's relevant here, since it's just a data object. Nobody's modifying its prototype.